lib/programmemory.cpp

```diff
@@ -233,6 +233,24 @@ static bool frontIs(const std::vector<MathLib::bigint>& v, bool i)
     return !i;
 }
 
+static bool isTrue(const ValueFlow::Value& v)
+{
+    if (v.isImpossible()) {
+        return v.intvalue == 0;
+    } else {
```

Comment:
We don't have readability-else-after-return enabled, but it's still redundant.

Reply:
I did not enable that because it's just too many changes and it doesn't really improve things much. In some cases it even breaks the if-else flow of some bugger chains since some scopes return and others don't which makes it feel "wrong".